### ISSUE TITLE
bootstrap producing events on two different instances pointing to same DB

### ISSUE THREAD

**Issue by jimymodi** [zendesk/maxwell#1026]
I am running maxwell with `--bootstrapper=async`, but the bootstrap is not running in separate thread.
`root     27653 21.7 10.6 7785000 1687000 pts/0 Sl   20:43   1:47 java -Dfile.encoding=UTF-8 -Dlog4j.shutdownCallbackRegistry=com.djdch.log4j.StaticShutdownCallbackRegistry -cp :bin/../lib/*:bin/../lib/kafka-clients/kafka-clients-0.11.0.1.jar com.zendesk.maxwell.Maxwell --config /etc/maxwell-1.13.3/configdb1.properties --bootstrapper=async`

**Comment by osheroff** [comment]
sorry, can you be more specific?

**Comment by jimymodi** [comment]
Apologies for a bad post in moment of distress. Here are the details.
We run 4 instance of maxwell on the same machine. Pointing to 3 different databases. 2 instance pointing to same database with different ID's. 
```
+-----------+------------------+-----------------+----------+------------+--------------+---------------------+
| server_id | binlog_file      | binlog_position | gtid_set | client_id  | heartbeat_at | last_heartbeat_read |
+-----------+------------------+-----------------+----------+------------+--------------+---------------------+
|    100149 | mysql-bin.262744 |        42209230 | NULL     | cpms_main  |         NULL |                   0 |
|    100149 | mysql-bin.262744 |        42329956 | NULL     | maxwell    |         NULL |                   0 |
+-----------+------------------+-----------------+----------+------------+--------------+---------------------+
```
Both the instance replicate the different set of tables and to two different topic of same kafka.
For e.g. Instance with client_id `maxwell` produces message to `topic_x` and client_id `cpms_main` produces to `topic_y`. 
Now, I started the bootstrapper for a table with the config of `cpms_main` client_id. The `bootstrap-insert` events where produced in both the topic `topic_x` as well as `topic_y`. 

Is this the expected behaviour, or a bug ?

**Comment by osheroff** [comment]
we fixed this back in v1.14.1.  what version are you running?

**Comment by jimymodi** [comment]
Ohh, We are on `maxwell-1.13.3`.
Thanks will get it upgraded.
